Assets/Scripts/Spawning/ResourceSpawning/SpawnArea.cs

```diff
@@ -0,0 +1,26 @@
+using Examen.Interactables.Resource;
+using Examen.Poolsystem;
+using FishNet.Object;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Examen.Spawning.ResourceSpawning
+{
+    public class SpawnArea : NetworkBehaviour
+    {
+        [SerializeField] private List<GameObject> _spawnedResources;
+
+        public List<GameObject> SpawnedResources { set => _spawnedResources.AddRange(value); }
+
+        [Server]
+        private void Start()
```

Comment:
Same here start cannot be tagged server. you could use `IsServer` this is the same as `isOwner` but just for server clients